inst/qml/Prophet.qml

```diff
@@ -0,0 +1,626 @@
+import QtQuick 			2.8
+import QtQuick.Layouts 	1.3
+import JASP.Controls 	1.0
+import JASP.Widgets 	1.0
+import JASP				1.0
+
+Form
+{
+	VariablesForm
+	{	
+		preferredHeight: 400 * preferencesModel.uiScale
+		AvailableVariablesList	{ name: "allVariablesList" }
+		AssignedVariablesList	{ name: "dependent"; title: qsTr("Dependent Variable");	suggestedColumns: ["scale"]; singleVariable: true													}
+		AssignedVariablesList	{ name: "time"; title: qsTr("Time"); suggestedColumns: ["nominal"]; singleVariable: true																	}
+		AssignedVariablesList	{ name: "changepoints"; title: qsTr("Changepoints"); suggestedColumns: ["scale"]; singleVariable: true											}
+		AssignedVariablesList	{ name: "capacity"; title: qsTr("Carrying Capacity"); suggestedColumns: ["scale"]; singleVariable: true; id: cap; enabled: growth.value === "logistic"		}
+		AssignedVariablesList	{ name: "minimum"; title: qsTr("Saturating Minimum"); suggestedColumns: ["scale"]; singleVariable: true; id: floor; enabled: growth.value === "logistic"	}
+		AssignedVariablesList	{ name: "covariates"; title: qsTr("Covariates"); suggestedColumns: ["scale"];																				}
+		AssignedVariablesList	{ name: "historyIndicator"; title: qsTr("History Indicator"); suggestedColumns: ["scale"]; singleVariable: true									}
```

Comment:
So I am thinking about this input, as it is quite unusual in JASP. I am not sure whether it will be obvious how to use it (the reason for my worry is that it took me a while to realise what this does). The obvious solution would be to bother @JorisGoosen and propose implementing a `boolean` type column, but I don't know whether it's a good idea. 

At the minimum, we could think if renaming this variable to something that everyone will understand would help. Something like `"Include in training (boolean 1/0)"` or something like that.

Other option would be to see how is similar stuff done in machine learning, maybe you could just have an input on what is the first date that is used for forecasting and not fitting, and then everything after that date is assumed to have `historyIndicator == 0`.

Reply:
I also think this is not really an optimal solution, yet, but it is the same the Machine Learning module has (but with a variable form instead of a dropdown). Having a cutoff value for predictions would also work, but is less flexible too. But if you think that it is easier to understand, I will change that! Otherwise, I will rename the variable form and maybe make the error messages more comprehesible.

Reply:
Let's stick to renaming this input. I saw that some error messages are already there, so that should be fine. Also remember to properly describe the expected input in the help files...